<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inform_settings_page</class>
 <widget class="QWidget" name="inform_settings_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="5,6,7">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>70</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget
{
background-color: rgb(235, 235, 235);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>
提醒方式</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="scaled_pixmap_label" name="voice">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="voice_text">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>语音播报</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="scaled_pixmap_label" name="music">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="music_text">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>特效音乐</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_6" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="scaled_pixmap_label" name="ad">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ad_text">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>振动</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget_2
{
background-color: rgb(235, 235, 235);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5" stretch="3,4,4">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>提醒时间设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_7" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,1,3,3,3,2,2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>距离药品有效期</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>天时，每天</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ComboLineEdit" name="lineEdit_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>提醒我</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="SwitchButton" name="widget_9" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>45</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_8" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="5,1,3,3,3,2,2">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>药品数量小于</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>时，每天</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ComboLineEdit" name="lineEdit_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>提醒我</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>170</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="SwitchButton" name="widget_10" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>45</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget_3
{
background-color: rgb(235, 235, 235);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,1,1">
      <item>
       <widget class="QWidget" name="widget_11" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,8,2">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>提醒间隔</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>449</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>10分钟</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15分钟</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>30分钟</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_12" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,8,2">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>音效</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>449</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_2">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>音效1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>音效2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>音效3</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_13" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,1,1,7">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>声音大小</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="scaled_pixmap_label" name="sound">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QSlider::groove:horizontal {
    height: 8px; /* 稍微增大轨道高度 */
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, 
                                stop: 0 rgb(124, 124, 124), 
                                stop: 1.0 rgb(72, 71, 71));
    border-radius: 4px; /* 轨道圆角 */
}

QSlider::handle:horizontal {
    /* 大圆形滑块 */
    background: qradialgradient(cx:0.5, cy:0.5, radius: 0.5, fx:0.5, fy:0.5,
                                stop:0.6 #f6f7fa, stop:0.7 #c4c4c4);
    width: 40px;   /* 滑块宽度 */
    height: 40px;  /* 滑块高度 */
    margin: -18px 0; /* 负边距使滑块超出轨道 */
    border-radius: 20px; /* 圆形滑块，半径为高度的一半 */
    border: 1px solid #5c5c5c;
}

/* 悬停效果 */
QSlider::handle:horizontal:hover {
    background: qradialgradient(cx:0.5, cy:0.5, radius: 0.5, fx:0.5, fy:0.5,
                                stop:0.6 #ffffff, stop:0.7 #d0d0d0);
}

/* 按下效果 */
QSlider::handle:horizontal:pressed {
    background: qradialgradient(cx:0.5, cy:0.5, radius: 0.5, fx:0.5, fy:0.5,
                                stop:0.6 #e0e0e0, stop:0.7 #a0a0a0);
}</string>
           </property>
           <property name="value">
            <number>50</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header location="global">switchcontrol.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>scaled_pixmap_label</class>
   <extends>QLabel</extends>
   <header location="global">scaled_pixmap_label.h</header>
  </customwidget>
  <customwidget>
   <class>ComboLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">mytimeedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
